src/utils/Browser.js

```diff
@@ -18,13 +18,35 @@ class Browser {
     }
 
     /**
-     * Returns whether browser is mobile.
+     * Returns the max touch points for touchscreen devices
+     * Helps in mocking out.
+     *
+     * Only supported in the following browsers:
+     * @see https://developer.mozilla.org/en-US/docs/Web/API/Navigator/maxTouchPoints
```

Comment:
See note about browser support. Particularly for the Mobile Safari use case, it is only available on iOS 13+.

Reply:
Removing function.